### PR TITLE
Propagate admin action lifespan to ResetCredentialDialog (#32547)

### LINKED ISSUE #32547: The set value ‘Default Admin-Initiated Action Lifespan’ has no effect on the ‘Credential Reset’ form
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

admin/ui

### Describe the bug

Even after override the value 'Default Admin-Initiated Action Lifespan' under 'Realm settings/Tokens', '12 Hours' is always preset in the form for the manual sending of a 'Credential Reset' mail.

This is probably caused by function **getActionTokenGeneratedByAdminLifespan()**, as a fixed value is always returned here:
https://github.com/keycloak/keycloak/blob/f2ff485b7fb96b6ea3ed598a6a3abff0f4b7bdda/model/jpa/src/main/java/org/keycloak/models/jpa/RealmAdapter.java#L611

However, the new lifespan value is used correctly when an email is initiated via the API.

### Version

25.0.4

### Regression

- [ ] The issue is a regression

### Expected behavior

Admin initiated actions form should use the override lifespan value.

### Actual behavior

Admin initiated actions form always set lifespan to 12 hours as default.

### How to Reproduce?

- Set 'Default Admin-Initiated Action Lifespan' at 'Realm settings / Tokens' to any value other than 12 hours
- Go to 'Users details / Credentials' for any user
- Click on 'Credentials Reset'

### Anything else?

Override value is correctly used by initiate an email via the API

PUT /admin/realms/{realm}/users/{id}/execute-actions-email

### PR DESCRIPTION
**Pull request by thomasdarimont**
Fixes #32547

### CODE REVIEW

**Review by edewit — CHANGES_REQUESTED** [review]
So the original code is not good. I think we don't need the default value on the controller when we already have it in the form

**Comment by thomasdarimont** [comment]
@edewit Thanks for taking a look! Can you be a bit more specific about what I should change?

**Comment by thomasdarimont** [comment]
I removed the lifespan constant value from the code.

File: js/apps/admin-ui/src/user/user-credentials/ResetCredentialDialog.tsx [hunk 1]
```diff
@@ -16,23 +17,22 @@ type ResetCredentialDialogProps = {
...
 }: ResetCredentialDialogProps) => {
   const { adminClient } = useAdminClient();
-
+  const { realmRepresentation: realm } = useRealm();
   const { t } = useTranslation();
   const form = useForm<CredentialResetForm>({
-    defaultValues: credResetFormDefaultValues,
+    defaultValues: {
+      actions: [],
+      lifespan: realm?.actionTokenGeneratedByAdminLifespan,
```
Comment (edewit) [inline js/apps/admin-ui/src/user/user-credentials/ResetCredentialDialog.tsx:33]: You either set the lifespan default value here

File: js/apps/admin-ui/src/user/user-credentials/LifespanField.tsx [hunk 2]
```diff
@@ -13,7 +12,7 @@ export const LifespanField = () => {
       units={["minute", "hour", "day"]}
       menuAppendTo="parent"
       controller={{
-        defaultValue: credResetFormDefaultValues.lifespan,
+        defaultValue: lifespan,
```
Comment (edewit) [inline js/apps/admin-ui/src/user/user-credentials/LifespanField.tsx:15]: or you set it here on the controller one of these can be removed

**Review by edewit — CHANGES_REQUESTED** [review]
only one of these is needed, setting both with the same value is overkill

**Comment by thomasdarimont** [comment]
Thanks for the hints @edewit. I tried to follow your suggestions in the latest commit.

**Review by edewit — APPROVED** [review]

**Pull request merged**